jsapi/src/main.js

```diff
@@ -708,7 +741,7 @@ lux.Map.prototype.addSearch = function(target) {
     'menuClass' : 'lux-search-suggestions',
     'source'    : function(term, suggest) {
       term = term.toLowerCase();
-      fetch(lux.searchUrl + 'limit=5&query=' + term).then(function(resp) {
+      fetch(lux.searchUrl + 'limit=5&layer=Adresse&query=' + term).then(function(resp) {
```

Comment:
In the initial offer it was asked that the in the API the search should be limited to addresses.
Let's open a different issue if it is expected to support more types.